doc/refexpr.rst

```diff
@@ -129,3 +129,47 @@ references in this structure, e.g.::
       myCons[i] = @addConstraint(m, x[i] >= i)
     end
 
+Vectorized operations
+^^^^^^^^^^^^^^^^^^^^^
+
+JuMP supports vectorized expressions and constraints for linear and quadratic models. Although this syntax may
+be familiar for users coming from other modeling languages, we caution that this syntax can be slow---especially
+for large operations---and that it often can destroy the sparse, algebraic structure that makes modeling
+languages useful. Nevertheless, the syntax often proves useful, for example in constraints involving small,
+dense matrix-vector products.
+
+Linear algebraic operators have been added to give meaning to expressions like ``A*x`` where ``A`` is a matrix
```

Comment:
I would just describe the functionality instead of using the past tense "have been added"